app/components/list-widget/template.hbs

```diff
@@ -1,8 +1,8 @@
 <table style="width: 100%;"class="list-widget widget-body">
-    {{#each data as |item|}}
+    {{#each data as |listItem|}}
```

Comment:
I see but the confusion is more in the component.js file and the naming in the transitonToFacet. But leave it for a fixes PR so the focus of this PR stays. This change can be merged though.